images/linux/scripts/installers/git-ftp.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+################################################################################
+##  File:  git-ftp.sh
+##  Desc:  Installs git-ftp
+################################################################################
+
+# Source the helpers for use with the script
+source $HELPER_SCRIPTS/document.sh
+
+set -e
```

Comment:
nitpick: It is better to put `set -e` before `source $HELPER_SCRIPTS/document.sh`